doc/api/single-executable-applications.md

```diff
@@ -43,10 +43,29 @@ tool, [postject][]:
    ```
 
 4. Create a copy of the `node` executable and name it according to your needs:
+
+   * On systems other than Windows:
+
    ```console
    $ cp $(command -v node) hello
    ```
 
+   * On Windows:
+
+   Using PowerShell:
+
+   ```console
+   $ cp (Get-Command node).Source hello.exe
+   ```
+
+   Using CMD:
+
+   ```console
+   for /F "tokens=*" %n IN ('where.exe node') DO @(copy "%n" hello.exe)
```

Comment:
```suggestion
   Using Command Prompt:

   ```console
   $ for /F "tokens=*" %n IN ('where.exe node') DO @(copy "%n" hello.exe)
```